universitat-basel-iberoromanistik.csl

```diff
@@ -0,0 +1,300 @@
+            </group>
+          </else>
+        </choose>
+      </if>
+    </choose>
+  </macro>
+  <macro name="locator-citation">
+    <group>
+      <text variable="locator"/>
+    </group>
+  </macro>
+  <macro name="published-date">
+    <choose>
+      <if type="article-newspaper speech interview" match="any">
+        <date variable="issued">
+          <date-part name="day" form="numeric" suffix=". "/>
+          <date-part name="month" form="long" suffix=" "/>
+          <date-part name="year" form="long"/>
+        </date>
+      </if>
+    </choose>
+  </macro>
+  <macro name="pages">
+    <text variable="page"/>
+  </macro>
+  <macro name="edition">
+    <choose>
+      <if is-numeric="edition">
+        <group>
+          <number variable="edition" form="numeric" suffix=". "/>
+          <text term="edition" form="short"/>
+        </group>
+      </if>
+      <else>
+        <text variable="edition" suffix="."/>
+      </else>
+    </choose>
+  </macro>
+  <macro name="container-prefix">
+    <choose>
+      <if type="chapter paper-conference" match="any">
+        <text term="in" suffix=":"/>
+      </if>
+    </choose>
+    <choose>
+      <if type="paper-conference" match="any">
+        <text variable="event" suffix=", "/>
+        <date variable="issued">
+          <date-part name="day" form="numeric" suffix=". "/>
+          <date-part name="month" form="long" suffix=" "/>
+          <date-part name="year" form="long" suffix=","/>
+        </date>
+      </if>
+    </choose>
+  </macro>
+  <macro name="container-title">
+    <choose>
+      <if type="chapter paper-conference" match="any">
+        <text macro="editor"/>
+      </if>
+    </choose>
+    <choose>
+      <if type="chapter" match="any">
+        <group delimiter=". " prefix=" ">
+          <text variable="container-title" text-case="title"/>
+          <text macro="number-of-volumes"/>
+        </group>
+      </if>
+      <else-if type="webpage post-weblog" match="any">
+        <text variable="container-title" suffix="."/>
+      </else-if>
+      <else-if type="interview" match="any">
+        <text value="Interview am" text-case="capitalize-first"/>
+      </else-if>
+      <else>
+        <text variable="container-title" font-style="italic" text-decoration="none"/>
+      </else>
+    </choose>
+  </macro>
+  <macro name="volume">
+    <group delimiter=" ">
+      <text variable="volume" form="short"/>
+      <label variable="volume" form="short"/>
+    </group>
+  </macro>
+  <macro name="number-of-volumes">
+    <group delimiter=" ">
+      <text variable="number-of-volumes"/>
+      <label variable="number-of-volumes" form="short"/>
+    </group>
+  </macro>
+  <citation et-al-min="3" et-al-use-first="1" disambiguate-add-year-suffix="true" collapse="year">
+    <sort>
+      <key macro="author"/>
+      <key variable="issued"/>
+    </sort>
+    <layout prefix="(" suffix=")" delimiter="; ">
+      <group delimiter=": ">
+        <group delimiter="">
```

Comment:
let's do this with delimiter rather than suffix (currently doesn't make a difference, but if e.g. we'll allow an author-only citation in the future it will.)